### DISCUSSION TITLE
Illuminance peaks aqara motion sensor

### DISCUSSION THREAD

**Discussion by jgarmar** [Koenkk/zigbee2mqtt#5351] [Devices]
![image](https://user-images.githubusercontent.com/6642578/102701069-76560600-4253-11eb-8f6d-bbe4d629117e.png)

Hi! Could someone tell me why I have those Illuminance peaks with the motion sensor aqara? It's always around 93, 92 luxes, I'm updated to the last version Thank you

**Comment by kevincaradant** [comment]
Hi

Not sure about Z2M faults.

Its look like your motion sensor is sometimes 'broken' and reports weird values. Mine don't do this. I check right now

![Screenshot_20201220-012416_Chrome](https://user-images.githubusercontent.com/6093175/102702382-5712a500-4262-11eb-928e-f9912da1cbc2.jpg)

Maybe your battery is weak? Did you try to change it? 
Do you have a good / enought signal with the co-ordinator / routers?

Edit: you are sure at these times, you didn't have powerful lights or something which can trigger the lux? 😁

**Comment by sjorge** [comment]
When i still used mine I also had peaks, it was because one of the lamps would hit the sensor.
So if there was motion and I turned on the lights it would have a huge peak.

**Comment by jgarmar** [comment]
Yes, but it is not the case, because the peaks occur when the room is dark and there is no movement ... I have asked for another sensor because it does not happen to me with the others that I have. thanks

**Comment by Koenkk** [comment]
Converted to a discussion since it's not a z2m issue.
